tidb-retool-integration.md

```diff
@@ -0,0 +1,22 @@
+# TiDB and Retool Integration
+1. Get TiDB Details & Whitelist Retool IP Address
+
+    You can get your database connection details from whoever set up the database. You'll also need to get them to [whitelist][1] the Retool IP address.
```

Comment:
What does "TiDB Details" mean? Would you please provide the specific information that is needed?